core/src/main/java/org/infinispan/topology/ClusterTopologyManagerImpl.java

```diff
@@ -197,6 +181,11 @@ public void stop() {
       }
    }
 
+   @Override
+   public ClusterManagerStatus getManagerStatus() {
```

Comment:
I wouldn't call it `getManagerStatus()` because it makes me think of DCM, maybe just `getStatus`